src/General/Icon/Icon.tsx

```diff
@@ -1,18 +1,38 @@
 import * as React from 'react';
 
 const Icon: React.FunctionComponent<Props> = props => {
-  const { children, color } = props;
+  const {
+    className,
+    children,
+    color,
```

Comment:
What do you think we set `currentColor` as default color prop here? That's what I mentioned that icon should be able to inherit color. https://caniuse.com/#search=currentColor

Reply:
@rosslo I think we can use currentColor if we find that most of the time, the icon color should be the same as the inherited color (from the parent container). I'm not sure if this is the case though. In any case, I think this change might lead to some breaking changes for our UI so maybe we can explore this in another issue?